plugins/ocm-backend/src/providers/ManagedClusterProvider.ts

```diff
@@ -91,6 +92,8 @@ export class ManagedClusterProvider implements EntityProvider {
           name: i.metadata.name,
           title: i.metadata?.labels?.name,
           annotations: {
+            [ANNOTATION_KUBERNETES_API_SERVER]:
+              i.spec?.managedClusterClientConfigs?.[0]?.url,
```

Comment:
Is this always located on index 0, isn't there a better marker?

Reply:
Not sure... The `managedClusterClientConfigs` key looks like this:
`managedClusterClientConfigs: [ [Object] ]`
where Object is `{ caBundle: val, url: val}`, I am not sure what other Objects can be inserted into `managedClusterClientConfigs` but the clusters which are managed by our OCM don't have anything else in here. I also don't see there anything that could be used for example with `.find()`. Do you have any ideas?

Reply:
I think you are right... Unless we try to pull this from `ManagedClusterInfo` (`.spec.masterEndpoint`). That may introduce unnecessary processing complexity. Let's keep it this way unless we face issues. Can you please add a comment that it can also be pulled from ^ with a link to this discussion?

Reply:
Resolved in latest commit :+1: